src/sentry/eventstore/models.py

```diff
@@ -210,7 +210,11 @@ def get_minimal_user(self) -> User:
                 {"id": user_id, "email": email, "username": username, "ip_address": ip_address}
             )
 
-        return self.get_interface("user")
+        user = self.get_interface("user")
+        if user is not None:
```

Comment:
As discussed, remove these changes and fix the test to only check id, email, username and ip

Reply:
Done in [071238f](https://github.com/getsentry/sentry/pull/66401/commits/071238f3ab6bd78283e0d2bcac153ade30e9c30c).